app/main/views/users.py

```diff
@@ -58,10 +72,8 @@ def list_users():
         single_user = user_query.filter(
             User.email_address == email_address.lower()
         ).first_or_404()
-
         return jsonify(
-            users=[single_user.serialize()],
-            links={}
+            users=[single_user.serialize()]
```

Comment:
Is this one we've missed or are we not doing it for a reason, maybe compatibility with the old one that sends an empty links reference? Think it's probably fine to update.

Reply:
This one is a little strange. If it doesn't find a user then it will 404. Whereas all our other list routes would just return an empty list. It's kind of like a mix between a single resource and a list resource. That's why I left it. Does that seem reasonable?

Reply:
Hmm. It's non-traditional, that's for sure, but I think it still qualifies as a `list_result_response`, and I think making it that (probably with a comment) would be better than leaving it.